dev/io.openliberty.microprofile.openapi.2.0.internal_fat_tck/publish/tckRunner/pom.xml

```diff
@@ -1,71 +1,118 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
-    Copyright (c) 2018 IBM Corporation and others. All rights reserved.
-    This program and the accompanying materials are made available under the 
-    terms of the Eclipse Public License v1.0 which accompanies this distribution, 
-    and is available at 
-        http://www.eclipse.org/legal/epl-v10.html 
-    Contributors: 
-        IBM Corporation - initial API and implementation
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
```

Comment:
Did you intend to get rid of the copyright?

Reply:
I didn't actually change this file. I deleted it and replaced it with the `pom.xml` from the `tck` subdirectory which didn't have a copyright comment.

I've always been slightly unsure of whether build files merit having copyright statements and they're a pain to update so I didn't add the comment to the file which didn't have one.